Riddler-Calvard/riddler_calvard.ipynb

```diff
@@ -0,0 +1,118 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "riddler_calvard.ipynb",
+      "provenance": [],
+      "collapsed_sections": []
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "L5R-tQX13TpI",
+        "outputId": "ae182f00-cc5c-418f-e94b-9ce87ba45993"
+      },
+      "source": [
+        "!pip install mahotas\n"
```

Comment:
```mahotas``` not allowed. 
Please red the Contributing Guidelines.

Reply:
@geekquad Riddler-calvard without mahotas?

Reply:
Our focus is to have various feature implementations of OpenCV in Python.